pkgs/rustic-rs/rustic/registry.yaml

```diff
@@ -98,9 +98,6 @@ packages:
             goarch: arm64
             replacements:
               linux: unknown-linux-gnu
-          - goos: windows
-            replacements:
-              arm64: arm64
```

Comment:
Removed an unnecessary override as the pre-built binaries for windows/arm64 don't exist.